src/NuGet.Core/NuGet.Configuration/Utility/SettingsUtility.cs

```diff
@@ -19,6 +19,7 @@ public static class SettingsUtility
         private const string HttpCacheEnvironmentKey = "NUGET_HTTP_CACHE_PATH";
         private const string PluginsCacheEnvironmentKey = "NUGET_PLUGINS_CACHE_PATH";
         private const string RepositoryPathKey = "repositoryPath";
+        private const string MaxHttpRequest = "maxHttpRequest";
```

Comment:
This name is a little ambiguous... could this somehow reflect concurrency?

Reply:
because this not really impact concurrency, we still have same operation level concurrency, but limit the http request number per source, NPM is using "maxSockets" for this feature, so I think "maxHttpRequest" may be more accurate for this flag.